assets/css/extended/share-buttons.css

```
@@ -0,0 +1,162 @@
+    transition: all 0.2s ease;
+    position: relative;
+    text-decoration: none;
+    padding: 0;
+    font-family: inherit;
+    font-size: inherit;
+}
+
+.share-btn svg {
+    width: 18px;
+    height: 18px;
+    fill: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)